src/manage_nsfs/manage_nsfs_validations.js

```diff
@@ -328,30 +328,33 @@ async function validate_bucket_args(config_fs, data, action) {
         if (!exists) {
             throw_cli_error(ManageCLIError.InvalidStoragePath, data.path);
         }
-        const account = await get_bucket_owner_account(config_fs, data.bucket_owner);
-        const account_fs_context = await native_fs_utils.get_fs_context(account.nsfs_account_config, data.fs_backend);
+
+        // bucket owner validations 
+        const owner_account = await get_bucket_owner_account(config_fs, undefined, data.owner_account);
```

Comment:
I think the variable name `owner_account` is a bit confusing because in the schema we have the same property name with string type (I understand that in the schema it should have been named `owner_account_id`... but we will probably not change it).